src/database/schema/ChatSchema.ts

```diff
@@ -147,7 +146,7 @@ class ChatSchema implements BaseDbSchema {
           ELSE 0 END AS is_continuous
       FROM 
       ${ChatSchema.getTableName()} A 
-      INNER JOIN ${UserSchema.getTableName()} B 
+      LEFT JOIN ${UserSchema.getTableName()} B 
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The database query has been changed from an INNER JOIN to a LEFT JOIN. This change will include all records from `ChatSchema` even if there is no matching record in `UserSchema`. Please ensure this is the intended behavior as it can potentially return more records than before.


<!-- This is an auto-generated comment by OSS CodeRabbit -->